src/git_client/git_cli.rs

```diff
@@ -60,14 +60,26 @@ impl GitClient for GitCli {
             .map(|&c| c as char)
             .collect::<String>();
 
-        let mut parts = out
-            .split(['/', '@', ':', '.'])
-            .collect::<Vec<&str>>();
-        parts.retain(|&s| !s.is_empty());
+        let is_https = match out.lines().next() {
+            Some(line) => line.contains("https"),
+            None => Err(eyre::eyre!("Unexpected git remote output"))
+                .suggestion("Please check the connection to your remote repository")?,
+        };
```

Comment:
This implies that every time only the first line is analysed, right? In the future would we like to analyse the `origin` remote, in case of multiple ones?

Reply:
Yes, that's exactly right. We'll implement cross repository PRs and handling multiple remotes in general in the future